schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    ID INTEGER,
```

Comment:
- [ ] [OPTIONAL] You could optionally make the id column not null and autoincremented as well as the primary key.